terraform.tfvars.example

```diff
@@ -26,7 +27,7 @@ db_username = "nerveshub"
 db_password = "123456"
 #db_allocated_storage = 20
 #db_instance_class = "db.t2.small"
-#db_engine_version = "11.4"
+db_engine_version = "12.10"
```

Comment:
Given the Postgres versioning policy and the use of `auto_minor_version_upgrade = true`, can we specify only the major version here? Otherwise Terraform is going to try to roll things back if applied again after a minor-version auto-upgrade.

Also, does it make the most sense for this variable to be overridden here by default? The variable is given a default value `11.4` when defined. If this needs to be 12, should that default be updated instead?

Reply:
I suspect it was the minor version of Postgres 11, not the use of 11, per se. It is still possible to provision 11.12 (and 9.6.22, for that matter). Or were you using an ARM instance class, which requires Postgres >=12.3?

Either way, I think this is a sensible update, but a) should better be updated as a default in `setup/variables.tf` and b) should be major-version only.

Reply:
I've just pushed an update that reverts my moves my changes from `terraform.tfvars.example` to `setup/variables.tf` with a major version only.